editor/app/components/slider.css

```diff
@@ -0,0 +1,140 @@
+/**
+ * Slider component
+ */
+.slider {
+  --hue: 240;
```

Comment:
@MiguelCastillo I've created a bunch of CSS variables here, but if you think it's a bit overhead, I can remove them.